ms-patches/microsoft-patch-index.yml

```diff
@@ -32,6 +31,7 @@ active-patches:
   # - ZZ_archive/ms-patches/feature-branch-name-1
   # ...
   # - ZZ_archive/ms-patches/feature-branch-name-N
+  # - ms-patches/use-all-windows-processor-groups

```

Comment:
you need to move the branch (a simple rename) and change this to ZZ_archive/ms-patches/use-all-windows-processor-groups